src/pages/Profile/EditMoto.js

```diff
@@ -0,0 +1,50 @@
+import axios from '../../../../axios';
+import { useHistory } from 'react-router-dom';
+import MotorcycleForm from '../MotorcycleForm';
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import useAuth from '../../../../hooks/useAuth';
+
+const EditMoto = props => {
+  const [auth] = useAuth();
+  const { id } = useParams();
+  const history = useHistory();
+  const [motorcycle, setMotorcycle] = useState(null);
+
+  const submit = async form => {
+    await axios.patch(`/motorcycles/${id}.json?auth=${auth.token}`, form);
+    history.push('/profil/motorcycles');
+  }
+
+  const fetchMoto = async () => {
+    const res = await axios.get(`/motorcycles/${id}.json`);
+    const motorcycleData = res.data;
+
+    delete(motorcycleData.user_id);

```

Comment:
Should be "userId". When you receive user_id from firebase you can transform it to userId and then in the rest places of the app please use "userId"